source/simulator/simulator_access.cc

```diff
@@ -147,14 +147,16 @@ namespace aspect
   bool
   SimulatorAccess<dim>::include_adiabatic_heating () const
   {
-    return simulator->parameters.include_adiabatic_heating;
+    std::vector<std::string> heating_models = simulator->heating_model_manager.get_active_heating_model_names();
+    return (std::find(heating_models.begin(), heating_models.end(), "adiabatic heating") != heating_models.end());
```

Comment:
Yes, I found that it was even used in some of the material models, so I thought it would be better to keep it like this (although you can get the same information from the heating model manager).